src/components/items-payment/index.scss

```diff
@@ -96,45 +84,37 @@
   display: flex;
   flex-direction: row-reverse;
 }
-
 /*
 .container {
   display: flex;
   flex-direction: column;
 }
-
 .container_nameCard {
   display: flex;
   flex-direction: column;
 }
-
 .container_dates {
   display: flex;
   justify-content: space-around;
 }
-
 .container_buttonP {
   display: flex;
   flex-direction: row-reverse;
 }
-
 .container_bank {
   display: flex;
   flex-direction: column;
 }
-
 .container_banks {
   display: grid;
   grid-template-columns: repeat(2, 1fr);
   grid-gap: 10px;
 }
-
 .check {
   grid-gap: 10px;
 }
-
 .container_payments {
   display: flex;
   flex-direction: row-reverse;
 }
-*/
\ No newline at end of file
+*/
```

Comment:
Please, once again, remove dead or commented code at least this case: in a next PR you will retake this approach for implementation, if this is not the case, just remove it, thx.

Reply:
Rodrigos's component